tests/integ/modin/types/test_timedelta_indexing.py

```diff
@@ -597,7 +597,7 @@ def loc_enlargement(key, item, df):
     # single row
     key = (native_pd.Timedelta("2 days 45 minutes"), slice(None, None, None))
 
-    with SqlCounter(query_count=1, join_count=1):
+    with SqlCounter(query_count=1, join_count=2):
```

Comment:
why we have extra join in this case?

Reply:
one join is from creating the df, one join is from `loc`